1-click.go

```diff
@@ -33,6 +34,17 @@ type OneClicksRoot struct {
 	List []*OneClick `json:"1_clicks"`
 }
 
+// InstallKubernetesApps represents a request required to install 1-click kubernetes apps
+type InstallKubernetesApps struct {
+	Slugs       []string `json:"addon_slugs"`
+	ClusterUUID string   `json:"cluster_uuid"`
+}
+
+// Message is the reponse of an kubernetes 1-click install request
+type Message struct {
+	MessageRsp string `json:"message"`
+}
```

Comment:
It might be more future proof and extensible to do something like:

```
type InstallKubernetesAppsResponse struct {
￼	Message string `json:"message"`
￼}
```

and then return the whole struct from `InstallKubernetes`. That way the response can potentially include new fields in the future (e.g. links to poll status?) without it being a breaking change to the godo API.

Reply:
Done 😄